rr_viz/src/rr_viz/widgets/mission_dashboard/mapping/slam_supervisor.py

```diff
@@ -256,6 +261,45 @@ def delete_map_slot(self):
                     print("failed calling slam delete map service")
                     print(trig_resp.message)
 
+    def save_map_image(self):
+        map_name = self.mapName.text()
+
+        if map_name !='':
+            if map_name.find('.'):
+                map_name = string.replace(map_name, '.', '_')
```

Comment:
What we really should be doing here is some proper sanity checking on file names. At least we should be doing some checking for `/` and this probably applies to everything else we do with any files.